.gitignore

```diff
@@ -23,4 +23,4 @@ docs-source/_build/**
 src/
 tests/fixtures/data/chunker/
 pytest_unit.xml
-cache/
+cache/
```

Comment:
why is it used? can the cache dir be moved to /tmp ?

Reply:
`cache` has always been defined as our `LIBRARY_OUT` dir, and it will usually be located inside the cloned repo directory, so need to add it to gitignore. Haven't tried using `tmp`

Reply:
I thought its defined in the cache dir of NLP Architect:
LIBRARY_OUT = Path(Path.home()) / "nlp-architect" / "cache"

Reply:
Yes, the line you sent defines `nlp-architect/cache` as the LIBRARY_OUT dir, as I wrote.